### DISCUSSION TITLE
Sphinx-Needs-Data-Explorer

### DISCUSSION THREAD

**Discussion by mi-parkes** [useblocks/sphinx-needs#1141] [Show and tell]
Hi UseBlocks,

I wanted to reach out and express my sincere appreciation for your work on the [Sphinx-Needs](https://github.com/useblocks/sphinx-needs) project. My colleagues and I have been using it for some time now, and it has been invaluable in managing the increasing amount of needs-data in our projects. However, as our projects grow and evolve, we've encountered limitations with static flows, tables, and other features.

To address these limitations, I've been working on a hobby project called  [Sphinx-Needs-Data-Explorer](https://pypi.org/project/sphinx-needs-data-explorer/). It's designed to complement Sphinx-Needs by offering dynamic and flexible ways to explore and manage needs-data.

I'd love to hear your thoughts and opinions on how Sphinx-Needs-Data-Explorer.

Thank you once again for your excellent work on Sphinx-Needs.

Best regards,
MP

**Comment by danwos** [comment]
Thanks for the hint.
It really looks promising and hopefully I find some time to try it in one of our projects.
I know at leat one other project, which has created something similar. But this is closed source :(

As such dynamic filtering and analysis is an ongoing topic, I'm glad that there is a solution right now :)

For everybody else: If you want to see it in action, just visit https://mi-parkes.github.io/sphinx-needs-data-explorer/ and press the `E` in the upper menu.
